sdcm/utils/common.py

```diff
@@ -1591,7 +1591,7 @@ def gce_download_dir(bucket, path, target):
 
     container = driver.get_container(container_name=bucket)
     dir_listing = driver.list_container_objects(container, ex_prefix=path)
-    for obj in dir_listing:
+    for obj in dir_listing[1:]:
```

Comment:
Add here a comment that explains why you do it

Reply:
done